packages/runtime/runtime-utils/src/summarizerNode.ts

```diff
@@ -149,10 +149,17 @@ function decodeSummary(snapshot: ISnapshotTree, logger: Pick<ITelemetryLogger, "
                         if (outstandingOps.length > 0 && newOutstandingOps.length > 0) {
                             const latestSeq = outstandingOps[outstandingOps.length - 1].sequenceNumber;
                             const newEarliestSeq = newOutstandingOps[0].sequenceNumber;
-                            assert(
-                                latestSeq < newEarliestSeq,
-                                `latestSeq exceeds newEarliestSeq in decodeSummary: ${latestSeq} >= ${newEarliestSeq}`,
-                            );
+                            if (newEarliestSeq <= latestSeq) {
+                                logger.sendTelemetryEvent({
+                                    eventName:"DuplicateOutstandingOps",
```

Comment:
You might want to set a flag and log it once with a counter of how many cases where hit in this loop